services/site/src/pages/404.tsx

```diff
@@ -13,13 +12,13 @@ const Custom404: React.FunctionComponent = () => {
         <title>404 Page not found | a11yphant</title>
         <meta name="robots" content="noindex,nofollow" />
       </Head>
-      <FullScreenLayout header={<Navigation />} footer={<Footer />}>
-        <main className={clsx("h-full box-border max-w-screen-3xl mx-auto")}>
-          <div className={clsx("mx-8 py-8 h-main", "lg:mx-24")}>
-            <ErrorWithIllustration error="Error 404" text="seems like you got lost in space" />
-          </div>
-        </main>
-      </FullScreenLayout>
+      <Navigation />
```

Comment:
Please do this exact change in the _error.tsx file as well. There the same bug occurs.